spec/controllers/articles_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+# require 'rails_helper'
```

Comment:
a to skąd? :)

Reply:
z pliku z zajęć 3 ale musiałam to zakomentować bo mi wywalało błędy